### PR TITLE
Rewrite arduino page to align with sbot

### PR DESCRIPTION
**Pull request by 8BitJosh**

### CODE REVIEW

File: docs/programming/arduino.md [hunk 1]
```diff
@@ -43,77 +44,76 @@ performing an action with that pin. You can read about the possible pin
...
-robot.arduino.pins[3].mode = GPIOPinMode.DIGITAL_INPUT_PULLUP
+robot.arduino.pins[3].mode = GPIOPinMode.INPUT_PULLUP
 ```
 
-### `GPIOPinMode.DIGITAL_INPUT`
+### Digital Input
 
-In this mode, the digital state of the pin (whether it is high or low)
-can be read.
+Digital inputs can be used to check the sate of a pin (whether it is high or low).
```
Comment (RealOrangeOne) [inline docs/programming/arduino.md:52]:
```suggestion
Digital inputs can be used to check the state of a pin (whether it is high or low).
```

File: docs/programming/arduino.md [hunk 2]
```diff
@@ -43,77 +44,76 @@ performing an action with that pin. You can read about the possible pin
...
 ```
 
-### `GPIOPinMode.DIGITAL_INPUT_PULLUP`
-
-Same as `GPIOPinMode.DIGITAL_INPUT`, but with an internal [pull-up
-resistor](https://learn.sparkfun.com/tutorials/pull-up-resistors)
-enabled.
+Some external switches may require a pull up resistor,
+`GPIOPinMode.INPUT_PULLUP`, is the same as INPUT but also enables an internal [pull-up
+resistor](https://learn.sparkfun.com/tutorials/pull-up-resistors).
```
Comment (RealOrangeOne) [inline docs/programming/arduino.md:63]:
```suggestion
Some external switches may require a pull up resistor.
`GPIOPinMode.INPUT_PULLUP`, is the same as `INPUT` but also enables an internal [pull-up
resistor](https://learn.sparkfun.com/tutorials/pull-up-resistors).
```

File: docs/programming/arduino.md [hunk 3]
```diff
@@ -43,77 +44,76 @@ performing an action with that pin. You can read about the possible pin
...
 ```
 
 !!! tip
-    The values are the voltages read on the pins, between 0 and 5.
+    The values are the voltages read on the pin, between 0 and 5.
 
-## Ultrasound Sensors
+### Ultrasound Sensors
 
 You can also measure distance using an ultrasound sensor from the arduino.
```
Comment (RealOrangeOne) [inline docs/programming/arduino.md:108]: Suggestion: We should explicitly note here that the return unit is millimetres.

**Review by RealOrangeOne — COMMENTED** [review]
Nice catch all around - added a few inline comments

File: docs/programming/arduino.md (hunk 3 above)
Comment (Adimote) [reply docs/programming/arduino.md]: Fixed in [cb70f69](https://github.com/sourcebots/docs/pull/217/commits/cb70f6955ee0b2bed3180f38138832923eeb4ed9)

File: docs/programming/arduino.md [hunk 4]
```diff
@@ -43,77 +44,76 @@ performing an action with that pin. You can read about the possible pin
...
-pin_value = robot.arduino.pins[AnaloguePin.A0].analogue_value
+pin_value = robot.arduino.pins[AnalogPins.A0].analog_value
 ```
 
 !!! tip
-    The values are the voltages read on the pins, between 0 and 5.
+    The values are the voltages read on the pin, between 0 and 5.
 
-## Ultrasound Sensors
+### Ultrasound Sensors
```
Comment (Adimote) [inline docs/programming/arduino.md:106]:
```suggestion
## Ultrasound Sensors
```

Ultrasound sensors aren't part of pin modes, so should be a new section in themselves

**Review by Adimote — APPROVED** [review]

**Pull request merged**